app/views/timed_service_price_policies/_table.html.haml

```diff
@@ -0,0 +1 @@
+= render "time_based_price_policies/table", price_policies: price_policies, url_date: url_date, cancellation: false, minimum_cost: false
```

Comment:
it'd be cool if we could find a way to not have to have this extra layer and just render this partial directly hmm